ui/lib/apps/SearchLogs/components/SearchHeader.tsx

```diff
@@ -121,13 +121,15 @@ export default function SearchHeader({ taskGroupID }: Props) {
 
       try {
         const result = await client.getInstance().logsTaskgroupPut(req)
-        const id = result.data.task_group?.id
+        const id = result?.data?.task_group?.id
         if (!id) {
-          throw new Error('Invalid server response')
+          Modal.error({
+            content: 'Invalid server response',
+          })
```

Comment:
The original way is simpler, avoid writing `Modal.error` twice :)

Reply:
But I think use Error to control the dataflow  is  not  a good  idea.

Reply:
I search this issue and find an answer: https://stackoverflow.com/questions/47015693/how-to-fix-throw-of-exception-caught-locally

But both OK for me, If you don't want this. I will revert this change later.

Reply:
Reverted.